src/02-common-principles.md

```diff
@@ -674,7 +674,85 @@ specificity and persistence.
 
 Several fields are designated for DOIs, for example, `DatasetDOI` in `dataset_description.json`.
 DOI values SHOULD be fully specified URIs such as `doi:10.18112/openneuro.ds000001.v1.0.0`.
-Bare DOIs such as `10.18112/openneuro.ds000001.v1.0.0` are [DEPRECATED][deprecated].
+Bare DOIs such as `10.18112/openneuro.ds000001.v1.0.0` are [DEPRECATED][].
+
+### BIDS URI
+
+To reference files in BIDS datasets, the following URI scheme may be used:
+
+```plain
+bids:<dataset-name>:<absolute-path>
+```
+
+The scheme component `bids` identifies a BIDS URI,
+which defines a `path` component of the form `<dataset-name>:<absolute-path>`.
+The `dataset-name` component is an identifier for a dataset,
+and the `absolute-path` component is a path, starting with a slash (`/`),
+followed by the location of a resource within the dataset.
```

Comment:
it seems absolute path simply means path relative to the root of the dataset. also to make it clear relative to unix systems, you could consider dropping the leading `/` so that there is no confusion with absolute locations on a file system.

Reply:
I kept this language, since that had been in @sappelhoff's version, but I'm okay moving away from that. Also okay with dropping the `/`.

Will wait a day or two to see if there are objections before moving forward with these changes.

Reply:
No objections against dropping the leading `/` from my side.